packages/datasets/mongodb_datasets/config.py

```diff
@@ -0,0 +1,17 @@
+"""Configuration utilities for mongodb_datasets package."""
+
+from pathlib import Path
+from dotenv import load_dotenv
+
+# Find the project root .env file by traversing up from this file
+# Structure: packages/datasets/mongodb_datasets/config.py -> ../../../.env
+PROJECT_ROOT = Path(__file__).parent.parent.parent.parent.parent
+ENV_PATH = PROJECT_ROOT / ".env"
```

Comment:
I assume this works but it's a bit confusing to me:

- The example (i.e. three levels of `..` from `config.py` would resolve to `packages/.env` which doesn't exist
- The actual code calls `parent` five times which doesn't match the example and seems to me like it would be the directory about the root `chatbot` repo.